### ISSUE TITLE
Vresion 0.1.6 works but 0.2.0 throws error Property __scanCodes doesn't exist

### ISSUE THREAD

**Issue by zzz08900** [rodgomesc/vision-camera-code-scanner#52]
At first I was using vision-camera-qrcode-scanner from npm(https://www.npmjs.com/package/vision-camera-qrcode-scanner), everything was alright.
Then I realized that package seems to be abandoned so I switched to vision-camera-code-scanner(package in this repo) and things went side ways really fast.

Background/package versions:
react-native@0.67.4
react-native-vision-camera@2.13.1
vision-camera-code-scanner@0.2.0

Platform: Android

Behavior: upon moutning the component with Camera and frameProcessor, the app crashes with error message "Property __scanCodes doesn't exist"

Version 0.1.6 from npm works fine though(no idea why).
babel.config.js is set according to the docs.

Any idea what was wrong (or an educated guess about where the problem might be)?

**Comment by zzz08900** [comment]
Hmm... things just got stranger.
I rebooted my computer, deleted node_modules, ran ./gradlew clean, re-installed everything and I suddenly I found version 0.2.0 is working again.

I tried the "wipe everything and re-install then rebuild" scheme multiple times (without the computer reboot) before opening this issue but it didn't help.

So my guess is Metro or  Gradle was holding some cache in their memory and they won't let the cache go no matter what.
Feels like I'm back in the 90s, when rebooting the computer was the first step to do upon running into strange issues.
